doc/source/examples/resolved_cfd-dem/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst

```diff
@@ -0,0 +1,271 @@
+		set pressure z =0.00001
+		set radius = 0.5
+	  end
+	end
+
+The hypershell around the boundary between ``refine mesh inside radius factor`` (r = 0.425) and ``refine mesh outside radius factor`` (r = 0.65) will initialy be refined twice (`initial refinement = 2`). 
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is solved at steady-state with 2 mesh adaptation.
+
+.. code-block:: text
+
+	subsection simulation control
+  	  set method                  = steady
+	  set number mesh adapt       = 2
+	  set output name             = sharp-carreau-output
+	  set output frequency        = 1
+	  set subdivision             = 1
+	end
+
+Mesh Adaptation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to generate the hypershell refinement zone around the immersed boundary, the ``mesh adaptation`` ``type`` must be set to ``kelly``. During both of the mesh refinement steps, 40% of the cells with be split in 8 (``fraction refinement   = 0.4``) using a velocity-gradient kelly operator.
+
+.. code-block:: text
+
+	subsection mesh adaptation
+	  set type                  = kelly
+	  set fraction coarsening   = 0.0
+	  set fraction refinement   = 0.4
+	  set fraction type	      	= number
+	  set frequency             = 1
+	  set max number elements   = 8000000
+	  set min refinement level  = 0
+	  set max refinement level  = 11
+	  set variable		      	= velocity
+	end
+
+Results
+---------------
+
+Using Paraview, the steady-state velocity profile and the pressure profile can be visualized by operating a *slice* along the xy-plane (z-normal) that cuts in the middle of the sphere (See `documentation <https://forgeanalytics.io/blog/creating-slices-in-paraview/>`_). 
+
+.. image:: images/velocity.png
+	:align: center
+
+.. image:: images/pressure.png
+	:align: center
+
+We can also see the viscosity profile throughout the domain, that is a function of the shear rate magnitude profile. Close to the particle, the shear rate is high which decreases the viscosity. 
+
+.. image:: images/viscosity.png
+	:align: center
+
+.. image:: images/shear_rate.png
+	:align: center
+
+We can notice that the viscosity rapidely reaches a plateau at :math:`\eta=0.063`. Given the parameters in the **Physical properties** section, the viscosity behavior should be given  by:
+
+.. image:: images/carreau.png
+	:align: center
+
+We get the following force applied on the particle for each of the mesh refinements.
+
+.. code-block:: text
+
+    particle_ID    T_x       T_y       T_z      f_x      f_y       f_z    
+	0 	-0.000008  0.000019 -0.000021 0.411248 0.000019  0.000038 
+	0 	 0.000000  0.000001 -0.000003 0.415503 0.000001  0.000002 
+	0 	-0.000000 -0.000000 -0.000000 0.424717 0.000002 -0.000002 
+          
+.. note:: 
+	Since analysing non-Newtonian flow, there is no known solution for the drag coefficient. For a Newtonian flow at Re = 50, the drag force would be 0.6165. Therefore, the drag force was decreased usgin a shear-thinning fluid.
```

Comment:
```suggestion
	Because this analysis concerns non-Newtonian flow, there is no known solution for the drag coefficient. For a Newtonian flow at Re = 50, the drag force would be 0.6165. Therefore, the drag force was decreased using a shear-thinning fluid, as expected.
```